discovery-infra/start_discovery.py

```diff
@@ -229,13 +229,13 @@ def set_cluster_vips(client, cluster_id, machine_net):
     client.update_cluster(cluster_id, cluster_info)
 
 
-def set_cluster_machine_cidr(client, cluster_id, machine_net):
+def set_cluster_machine_cidr(client, cluster_id, machine_net, set_vip_dhcp_allocation=True):
     cluster_info = client.cluster_get(cluster_id)
-    cluster_info.vip_dhcp_allocation = True
-    cluster_info.machine_network_cidr = machine_net.cidr_v4
+    if set_vip_dhcp_allocation:
+        cluster_info.vip_dhcp_allocation = True
+    cluster_info.machine_network_cidr = machine_net.cidr_v6 if machine_net.has_ip_v6 and not machine_net.has_ip_v4 else machine_net.cidr_v4
```

Comment:
Can be simplified as `machine_net.cird_v4 if machine_net.has_ip_v4 else machine_net.cidr_v6`

Reply:
It is not the same